Includes unistd.h to enable rmdir for IlmImfTest.
Will not send upstream.

--- openexr-3.1.5/src/test/OpenEXRUtilTest/tmpDir.h	Thu Apr  7 08:52:25 2022
+++ openexr-3.1.5/src/test/OpenEXRUtilTest/tmpDir_.h	Sat Jun  4 09:56:16 2022
@@ -15,6 +15,7 @@
     #define rmdir _rmdir
 #else
     #include <sys/stat.h> // for mkdir
+	#include <unistd.h> // for rmdir
     #define IMF_TMP_DIR "/var/tmp/"
     #define IMF_PATH_SEPARATOR "/"
 #endif
--- openexr-3.1.5/src/test/OpenEXRTest/tmpDir.h	Thu Apr  7 08:52:25 2022
+++ openexr-3.1.5/src/test/OpenEXRTest/tmpDir_.h	Sat Jun  4 09:57:54 2022
@@ -15,6 +15,7 @@
     #define rmdir _rmdir
 #else
     #include <sys/stat.h> // for mkdir
+	#include <unistd.h> // for rmdir
     #define IMF_TMP_DIR "/var/tmp/"
     #define IMF_PATH_SEPARATOR "/"
 #endif
